src/libpriv/rpmostree-core.c

```diff
@@ -4017,6 +4022,10 @@ rpmostree_context_assemble (RpmOstreeContext      *self,
           if (!rpmostree_passwd_complete_rpm_layering (tmprootfs_dfd, error))
             return FALSE;
         }
+
+      /* And clean up var/tmp, we don't want it in commits */
+      if (!glnx_shutil_rm_rf_at (tmprootfs_dfd, "var/tmp", cancellable, error))
```

Comment:
This should be outside this if-else though, right? Otherwise if we're not actually overlaying anything, it'll still be committed.

Reply:
Yep, good catch, thanks!  Fixup :arrow_down: